src/lib/components/AppTagInput/AppTagInput.tsx

```diff
@@ -0,0 +1,37 @@
+import { Field, FieldLabel, TagsInput, Text } from "@chakra-ui/react";
+
+interface TagInputProps {
+  title?: string;
+  disabled?: boolean;
+  value?: string[];
+  onChange: (value: string[]) => void;
+  border?: string;
+  bg?: string;
+  size?: "sm" | "md" | "lg";
+}
+
+const AppTagInput: React.FC<TagInputProps> = ({ title, disabled = false, onChange, value }) => {
+  return (
+    <Field.Root disabled={disabled} data-test="taginput-root">
+      {title && (
+        <FieldLabel data-test="taginput-label">
+          <Text as="b">{title}</Text>
+        </FieldLabel>
+      )}
+      <TagsInput.Root
+        value={value ?? []}
+        onValueChange={details => {
+          onChange(details.value);
+        }}
+      >
+        <TagsInput.Label>Tags</TagsInput.Label>
+        <TagsInput.Control>
+          <TagsInput.Items />
+          <TagsInput.Input data-test="taginput-input" placeholder="Add tag..." />
```

Comment:
placeholders must be transalted too. being a lib component, this is to be in a lib namespace not in the default translation one.